addons/gut/spy.gd

```diff
@@ -62,7 +62,7 @@ func was_called(variant, method_name, parameters=null):
 	return to_return
 
 
-func get_call_parameters(variant, method_name, index=-1):
+func get_call_parameters(variant, method_name, index=-1) -> Variant:
```

Comment:
`Array`, optionally `Array[Variant]`, I'm fine with either, we just have to be consistent.